### PR TITLE
feat: return client config as the second value in GenerateConfiguration

### PR DESCRIPTION
**Pull request by Unix4ever**
To be used in interactive installer to output the node client
configuration to a file.

Signed-off-by: Artem Chernyshev <artem.0xD2@gmail.com>

### CODE REVIEW

**Review by andrewrynhard — APPROVED** [review]
Curious how you see this working with the join process in the future. Do we always generate both?

File: internal/pkg/configuration/configuration.go [hunk 1]
```diff
@@ -86,7 +87,20 @@ func Generate(ctx context.Context, in *machine.GenerateConfigurationRequest) (re
...
+		}
+
+		taloscfgBytes, err = talosconfig.Bytes()
+
+		if err != nil {
+			return nil, err
+		}
+
+		reply = &machine.GenerateConfigurationResponse{
+			Data: [][]byte{cfgBytes, taloscfgBytes},
```
Comment (andrewrynhard) [inline internal/pkg/configuration/configuration.go:102]: It feels fragile to have the configs in a slice. Ordering becomes important at that point.

Comment (smira) [reply internal/pkg/configuration/configuration.go:102]: probably deserves a field on its own

Comment (Unix4ever) [reply internal/pkg/configuration/configuration.go:102]: :heavy_check_mark:

**Comment by Unix4ever** [comment]
> Curious how you see this working with the join process in the future. Do we always generate both?

We can generate both all the time. But actually currently we don't have the code for join config generation.

**Review by smira — APPROVED** [review]

**Pull request merged**